cadctap/intTest/test_all.py

```diff
@@ -8,6 +8,10 @@
 from astropy.io import fits
 import tempfile
 import numpy as np
+import cadctap
+from cadcutils.net import Subject
+from six import BytesIO
```

Comment:
Why not use the Python provided `BytesIO`?

Reply:
six.BytesIO and six.StringIO are compatible with both Python 2 and 3.